Formula/libb64.rb

```diff
@@ -0,0 +1,36 @@
+class Libb64 < Formula
+  desc "Base64 encoding/decoding library"
+  homepage "https://libb64.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/libb64/libb64/libb64/libb64-1.2.src.zip"
+  sha256 "343d8d61c5cbe3d3407394f16a5390c06f8ff907bd8d614c16546310b689bfd3"
+
+  def install
```

Comment:
```suggestion
  keg_only :provided_by_macos

  def install
```

Reply:
This can be skipped given the binary isn't installed.